### ISSUE TITLE
wrong number of words in seed phrase

### ISSUE THREAD

**Issue by vargrant** [lukechampine/user#20]
I am trying to use seed phrase from SIAD

export WALRUS_SEED=.....
SEED PHRASE IS EXACTLY CORRECT

./user form 7962d480 8SC 278793 
Using WALRUS_SEED environment variable
wrong number of words in seed phrase

**Comment by vargrant** [comment]
Here without WALRUS_SEED but result is same

./user form 7962d480 8SC 278793
Seed: 
wrong number of words in seed phrase

It's seems WALRUS SEED is SEED NOT FROM SIAD ?

**Comment by vargrant** [comment]
Ok - understood.  When I disabled walrus and shard servers in config.toml then have next result

./user form 7962d480 8SC 278793 
Contract formation failed: FormContract: ReadResponse: communication error: renter proposed a file contract with a too-long duration

**Comment by vargrant** [comment]
Better to rename this parameter - must be depend from settings. Walrus Seed or SIAD Seed ...

**Comment by lukechampine** [comment]
Yeah, `walrus` seeds are different from `siad` seeds. To generate a `walrus` seed, run `walrus-cli seed`.
